Readme_orig.md

```diff
@@ -0,0 +1,43 @@
+# Today's Objective: Hack the Résumé
```

Comment:
quick tip: having this as your "actual" readme will show potential employers the problem set, so they can understand your code better!